README.md

```diff
@@ -136,38 +119,18 @@ This will output queries like the following:
 
 ```
 sqlite> SELECT * FROM association_table;
-creation_time  mod_time    deleted     user_name   admin_level  account     shares      max_jobs    max_wall_pj
+creation_time  mod_time    deleted     username    admin_level  bank        shares      max_jobs    max_wall_pj
 -------------  ----------  ----------  ----------  -----------  ----------  ----------  ----------  -----------
-1589225734     1589225734  0           fluxuser    1            acct        10          100         60  
+1605309320     1605309320  0           fluxuser    1            foo         1           1           60       
 ```
 
-The second way is to use flux-accounting's command line arguments. You can pass in a path to the database file, or be in the same directory where the database file (**FluxAccounting.db**) is located:
-
-```
-$ flux account -h
-
-usage: flux-account.py [-h] {view-user,add-user,delete-user,edit-user} ...
-
-Description: Translate command line arguments into SQLite instructions for the
-Flux Accounting Database.
-
-positional arguments:
-  {view-user,add-user,delete-user,edit-user}
-                        sub-command help
-    view-user           view a user's information in the accounting database
-    add-user            add a user to the accounting database
-    delete-user         remove a user from the accounting database
-    edit-user           edit a user's value
-
-optional arguments:
-  -h, --help            show this help message and exit
-```
+The second way is to use flux-accounting's command line arguments. You can pass in a path to the database file, or be in the same directory where the database file (**FluxAccounting.db**) is located.
```

Comment:
The "be in the same directory" will not be true after #73 is merged.  If you don't provide an explicit path with `-p`, it will default to the "compiled-in" path of `${prefix}/var/FluxAccounting.db`.